engine/src/main/java/org/terasology/config/PlayerConfig.java

```diff
@@ -46,6 +50,18 @@ public void setColor(Color color) {
         this.color = color;
     }
 
+    public Float getHeight() { return height; }
+
+    public void setHeight(Float height) {
+        this.height = height;
+    }
+
+    public Float getEyeHeight() { return eyeHeight; }
+
+    public void setEyeHeight(Float eyeHeight) {
+        this.eyeHeight = eyeHeight;
```

Comment:
would it make sense to restrict eye height to be lower than player height? should we check this?